packages/chat/logo.ts

```diff
@@ -1,52 +0,0 @@
-import chalk from 'chalk'
```

Comment:
need to backup this

Reply:
nothing is ever truly gone in our memories, or in git.